spokestack/models/tensorflow.py

```diff
@@ -25,18 +25,21 @@ def __init__(self, model_path: str, **kwargs: Any) -> None:
         self._output_details = self._interpreter.get_output_details()
         self._interpreter.allocate_tensors()
 
-    def __call__(self, inputs: List[np.ndarray]) -> List[np.ndarray]:
```

Comment:
a simpler way to do this would be to accept the parameter as *args, then it's always a list, and clients can either pass the input tensors directly or as *list: 
```
model(tensor1, tensor2)
model(*tensors)
```